x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -112,6 +124,31 @@ export const PolicyList = memo(() => {
       },
     }
   );
+
+  const outdatedManifestsCallOut = () => {
```

Comment:
I would suggest using `useMemo` here to avoid unnecessary renders.

Reply:
Added